source/_integrations/matter.markdown

```diff
@@ -16,6 +16,7 @@ ha_codeowners:
 ha_domain: matter
 ha_platforms:
   - binary_sensor
+  - climate
```

Comment:
Update the category list too.

Reply:
adjusted

Reply:
oops, forgot to hit the push button and went for lunch :-)